qml/MonitorMenuBar.qml

```diff
@@ -75,6 +77,13 @@ MenuBar {
             text: (panels.showLeftSidebar) ? "Hide Left sidebar" : "Show Left Sidebar"
             onTriggered: leftSidebarHidden()
         }
+        Action {
```

Comment:
I think this is the most important action so I'd move it to the top of this menu and add a MenuSeparator right after it

Reply:
:+1: